lib/secretsService.js

```diff
@@ -0,0 +1,47 @@
+const apiUrl = '/api' // FIXME
+
+export default class SecretsService {
```

Comment:
we should change the design of this to avoid having to provide the user. user is in context, we can use context to make sure the service we use to access the API has it without providing it explicitly

Reply:
Agreed! I'm going with a different design moving forward.